test/test-tcp-try-write.c

```diff
@@ -57,6 +57,10 @@ static void connect_cb(uv_connect_t* req, int status) {
       bytes_written += r;
       break;
     }
+
+    buf = uv_buf_init("", 0);
+    r = uv_try_write((uv_stream_t*) &client, &buf, 1);
+    ASSERT(r == 0 || r == UV_EAGAIN);
```

Comment:
Also, you should probably move this up a few lines, else it's not going to get reached when the previous uv_try_write() call succeeded.